deploy/gk-deploy

```diff
@@ -40,20 +43,26 @@ Arguments:
 Options:
   -g, --deploy-gluster
               Deploy GlusterFS pods on the nodes in the topology that contain
-              brick devices. Default is to not deploy GlusterFS, thus assuming
-              that has already been done.
+              brick devices. If the --abort flag is also specified, this flag
+              indicates that all GlusterFS pods and deployments should be
+              deleted as well. Default is to not handle GlusterFS deployment
+              or removal.
 
   -t TEMPLATES, --templates_dir TEMPLATES
               Location of directory containing the heketi templates for
               OpenShift. Default is '${TEMPLATES}'.
 
+  -n NAMESPACE, --namespace NAMESPACE
```

Comment:
@jarrpa, i see @humblec's point, but I agree that it would be a good first approach to have a check whether the namespace exists and fail if it does not.